### ISSUE TITLE
Error raised when start Vim

### ISSUE THREAD

**Issue by BaksiLi** [sillybun/setbreakpoints_python#1]
```Error detected while processing /Users/mac/.vim/bundle/setbreakpoints_python/plugin/setbreakpoints_python.vim:
line    4:
E319: Sorry, the command is not available in this version: python3 import sys
line    5:
E319: Sorry, the command is not available in this version: python3 import vim
line    6:
E319: Sorry, the command is not available in this version: python3 import re
line    7:
E319: Sorry, the command is not available in this version: python3 sys.path.append(vim.eval('expand("<sfile>:h")'))
```   

I used Vundle to install this bundle, while configured as what the instruction said.

Thanks!

**Comment by sillybun** [comment]
What is your `vim --version`?

**Comment by sillybun** [comment]
Maybe your vim is compiled without `+python3` support

**Comment by LiSongcheng** [comment]
> Maybe your vim is compiled without `+python3` support

Thanks for reply. 
I ran `import sys; print(sys.version)` in vim, resulting in the following:

```
  3.7.0 (default, Jun 28 2018, 07:39:16)
  [Clang 4.0.1 (tags/RELEASE_401/final)]
```

So I reckon it's the problem with my vim settings? e.g. conflict with some Plugin?

**Comment by BaksiLi** [comment]
> Maybe your vim is compiled without `+python3` support

```
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Aug 17 2018 17:08:16)
Included patches: 1-503, 505-680, 682-1283
Compiled by root@apple.com
Normal version without GUI.  Features included (+) or not (-):
+acl             +file_in_path    -mouse_sgr       +tag_old_static
-arabic          +find_in_path    -mouse_sysmouse  -tag_any_white
+autocmd         +float           -mouse_urxvt     -tcl
-balloon_eval    +folding         +mouse_xterm     -termguicolors
-browse          -footer          +multi_byte      -terminal
+builtin_terms   +fork()          +multi_lang      +terminfo
+byte_offset     -gettext         -mzscheme        +termresponse
+channel         -hangul_input    +netbeans_intg   +textobjects
+cindent         +iconv           +num64           +timers
-clientserver    +insert_expand   +packages        +title
-clipboard       +job             +path_extra      -toolbar
+cmdline_compl   +jumplist        -perl            +user_commands
+cmdline_hist    -keymap          +persistent_undo +vertsplit
+cmdline_info    +lambda          +postscript      +virtualedit
+comments        -langmap         +printer         +visual
-conceal         +libcall         -profile         +visualextra
+cryptv          +linebreak       +python/dyn      +viminfo
+cscope          +lispindent      -python3         +vreplace
+cursorbind      +listcmds        +quickfix        +wildignore
+cursorshape     +localmap        +reltime         +wildmenu
+dialog_con      -lua             -rightleft       +windows
+diff            +menu            +ruby/dyn        +writebackup
+digraphs        +mksession       +scrollbind      -X11
-dnd             +modify_fname    +signs           -xfontset
-ebcdic          +mouse           +smartindent     -xim
-emacs_tags      -mouseshape      +startuptime     -xpm
+eval            -mouse_dec       +statusline      -xsmp
+ex_extra        -mouse_gpm       -sun_workshop    -xterm_clipboard
+extra_search    -mouse_jsbterm   +syntax          -xterm_save
-farsi           -mouse_netterm   +tag_binary
```

**Comment by BaksiLi** [comment]
I have now figured out this. This is because my preinstalled Vim is **NOT compiled with python3**, as above comment shown:- `-python3`.

_N.B._ To check if your vim is installed with python3, try this command in the vim: `:echo has('python3')`, which returns a Boolean value of status of installation. In my case, it was 0.

After googling this problem, I have gone to the conclusion that, you must 'reinstall' Vim with the option of python3. The following is what I have done, hope this provides a little help to the late-comers:

```
brew uninstall vim
```

And for some reasons my brew doesn't take the option of `--with-python3` (checked from `brew info vim`), so I used mercurial (see the [official instruction using mecurial](https://www.vim.org/mercurial.php), git should work as well?).
```
brew install mercurial
hg clone https://bitbucket.org/vim-mirror/vim && cd vim
make distclean
```
if not writable, try this: `sudo chown -R $USER /usr/local/`

now you are ready to configure:
```
./configure \
--enable-python3interp=yes \
--with-python3-config-dir=/Users/mac/anaconda3/lib/python3.7/config-3.7m-darwin  # check this line and replace the path with your python3 config

make
```
add `--prefix=/usr/local` if you want to have a different path.

after compiling, try with the new vim by:
```
./vim
```
and type `:echo has('python3')`, now it should gives 1. Don't desperate if it doesn't, because sometimes it is not working before you install it, for some reason.

finally, install this vim by:
```
sudo make install
```
check now:

```
vim --version | grep python
```

Voila! Pefection!

ref: https://github.com/Valloric/YouCompleteMe/wiki/Building-Vim-from-source
